guvnor-services/guvnor-services-backend/src/main/java/org/guvnor/common/services/backend/metadata/attribute/GeneratedAttributesUtil.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.guvnor.common.services.backend.metadata.attribute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class GeneratedAttributesUtil {
+
+    private GeneratedAttributesUtil() {
+    }
+
+    public static Map<String, Object> cleanup(final Map<String, Object> _attrs) {
+        final Map<String, Object> attrs = new HashMap<>(_attrs);
+
+        attrs.replace(GeneratedAttributesView.GENERATED_ATTRIBUTE_NAME, null);
```

Comment:
@ederign Just followed the same pattern we use for other attribute types (e.g. https://github.com/kiegroup/guvnor/pull/473/files/683a7c522f14cfcef9869bcf0016d30a9c5eedab#diff-a24efd5c5afe6306ed0e2039004af29aR133), i.e. remove mappings for keys defined by given key. I'll add a comment with the explaination.